v3/tasks/20250828-v3-hyperevm-rate-provider/test/task.fork.ts

```diff
@@ -0,0 +1,39 @@
+import hre from 'hardhat';
+import { Contract } from 'ethers';
+import { describeForkTest, getForkedNetwork, Task, TaskMode } from '@src';
+import { expect } from 'chai';
+
+describeForkTest('V3-HyperEVMRateProviderFactory', 'hyperevm', 12336700, function () {
+  let task: Task;
+  let factory: Contract;
+
+  const FACTORY_NAME = 'HyperEVMRateProviderFactory';
+  const VERSION_NUMBER = 1;
+  const VAULT = '0xbA1333333333a1BA1108E8412f11850A5C319bA9';
+
+  before('run task', async () => {
+    task = new Task('20250828-v3-hyperevm-rate-provider', TaskMode.TEST, getForkedNetwork(hre));
+    await task.run({ force: true });
+    factory = await task.deployedInstance(FACTORY_NAME);
+  });
+
+  it('checks vault', async () => {
+    const vault = await factory.getVault();
+    expect(vault).to.be.eq(VAULT);
+  });
```

Comment:
```suggestion
  before('setup contracts', async () => {
    const vaultTask = new Task('20241204-v3-vault', TaskMode.READ_ONLY, getForkedNetwork(hre));
    vault = await vaultTask.deployedInstance('Vault');
  });

  it('checks vault', async () => {
    const factoryVault = await factory.getVault();
    expect(factoryVault).to.be.eq(vault.address);
  });
```
Could do this instead of hard-coding the Vault address, since it is deployed.